iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -89,12 +89,8 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
         }
 
         self.dataSource.handleLoadMoreMessages = { [unowned self] cid in
-            Task {
-                self.conversationController?.loadPreviousMessages()
-            }
+            self.conversationController?.loadPreviousMessages()
         }
-        
-        self.collectionLayout.delegate = self
```

Comment:
The collection layout no longer has a delegate. Unfortunately, the delegate would sometimes fire before the cell had appeared which caused problems.